### DISCUSSION TITLE
Real mode support for Win16 programs for Win1.x/2.x?

### DISCUSSION THREAD

**Discussion by Torinde** [FEX-Emu/FEX#4234] [Ideas]
Is it feasible to add support for Win16 Win1.x/2.x programs that use CPU real mode - like OTVDM?
- https://bugs.winehq.org/show_bug.cgi?id=26715
- https://github.com/otya128/winevdm/issues/1084

**Comment by Sonicadvance1** [comment]
We're not planning on adding 16-bit support.
